.reuse/dep5

```diff
@@ -0,0 +1,47 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Files: README.md
+       *.lock
+       *.md
+       data/*
+       doc/*
+       sw/cheri/sim_boot_stub/sim_boot_stub
+       vendor/lowrisc_ip/*
+       vendor/lowrisc_ibex/*
+       vendor/patches/*
+Copyright: lowRISC contributors
+License: Apache-2.0
+
+Files: vendor/display_drivers/*
+Copyright: 2022 Douglas Reis
+License: Apache-2.0
+
+Files: vendor/cheriot_safe/*
+Copyright: Microsoft Corporation
+License: Apache-2.0
+
+Files: vendor/pulp_riscv_dbg/*
+Copyright: 2020 ETH Zurich and University of Bologna
+License: SHL-0.51
+
+Files: vendor/open_hbmc/*
+Copyright: © 2020-2022, Vaagn Oganesyan <ovgn@protonmail.com>
+License: Apache-2.0
+
+Files: vendor/newae/crypto/tiny-AES128-C/*
+Copyright: tiny-AES-c contributors
+License: Unlicense
+
+Files: vendor/newae/simpleserial*
+       vendor/newae/hal/hal.h
+       vendor/newae/basic-passwdcheck/basic-passwdcheck.c
+Copyright: 2012-2017 NewAE Technology Inc
+License: GPL-3.0-or-later
```

Comment:
Looks like we have some license compatibility issue. We should either remove the dependency, or ask NewAE to relicense them to a permissive license.

Reply:
Please can we still get this PR in to avoid more licensing issues in the future.

*@GregAC for visibility into the licensing problem*